pkg/providers/amifamily/ami.go

```diff
@@ -219,11 +219,22 @@ func (p *Provider) fetchAMIsFromEC2(ctx context.Context, amiSelector map[string]
 	p.ec2Cache.SetDefault(fmt.Sprint(hash), output.Images)
 	amiIDs := lo.Map(output.Images, func(ami *ec2.Image, _ int) string { return *ami.ImageId })
 	if p.cm.HasChanged("amiIDs", amiIDs) {
-		logging.FromContext(ctx).With("ami-ids", amiIDs).Debugf("discovered images")
+		logging.FromContext(ctx).With("ami-ids", concatenateAmiIDs(amiIDs)).Debugf("discovered images")
 	}
 	return output.Images, nil
 }
 
+func concatenateAmiIDs(amisIds []string) string {
+	var amisIdsString string
```

Comment:
```suggestion
	var amiString string
```

nit: It's easier to read the variable name this way.